src/mudlet.pro

```diff
@@ -86,6 +68,72 @@ macx:QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.10
 QT += network opengl uitools multimedia gui concurrent
 qtHaveModule(gamepad): QT += gamepad
 
+############################# TEMPORARY TESTING PART ###########################
```

Comment:
How long do we want to keep this temporary part?

Reply:
Ah, for a bit - enough perhaps so I can take a look again at my work on Cygwin and FreeBSD ports (and those will be downloading release version tarballs to work from {possibly applying release specific patches until any changes that can _reasonably_ be accommodated into the main codebase}) so I would like it to be present in the next release so I can work out precise platform specific logic), perhaps? It shouldn't produce any visible output in __normal__ uses case now, I think.